admin/class-create-block-theme-admin.php

```diff
@@ -331,6 +336,18 @@ function add_theme_json_to_local ( $export_type ) {
 		);
 	}
 
+	function add_theme_json_variation_to_local ( $export_type, $theme ) {
+		$variation_path = get_stylesheet_directory() . DIRECTORY_SEPARATOR . 'styles' . DIRECTORY_SEPARATOR;
+		if ( ! file_exists( $variation_path ) ) {
+			mkdir( $variation_path, 0755, true );
+		}
+
+		file_put_contents(
+			$variation_path . '/' . $theme['variation'] . '.json',
```

Comment:
Do you think we should replace spaces in the variation name with hyphens? e.g. changing Dark Blue.json to dark-blue.json? I think most file systems will handle spaces anyway, but it's probably best practice to remove them.

Reply:
`$variation_path` already ending with `DIRECTORY_SEPARATOR`. extra `/` can be removed.